### PR TITLE
Update Korean translations

### PR DESCRIPTION
**Pull request by microbit-matt-hillsdon**

### CODE REVIEW

**Review by microbit-robert — COMMENTED** [review]
I'm unsure about the resulting mix of single and double quotes, i.e., booleanIsLowerCase vs every other string. Should retain double quotes for this?
image

File: packages/pyright-internal/src/localization/simplified.nls.ko.json [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
-        "argPositional": "위치 인자가 필요함",
-        "argPositionalExpectedOne": "위치 인자가 1개 필요함",
-        "assignmentTargetExpr": "표현식은 할당 대상이 될 수 없음",
-        "booleanIsLowerCase": "\"{name}\"(이)가 정의되지 않았습니다. \"{booleanName}\"입니까?",
+        "argMorePositionalExpectedCount": "위치 인자가 {expected}개 더 필요합니다.",
+        "argMorePositionalExpectedOne": "위치 인자를 1개 더 예상했습니다.",
+        "argPositional": "위치 인자가 필요합니다.",
+        "argPositionalExpectedOne": "위치 인자가 1개 필요합니다.",
+        "assignmentTargetExpr": "수학식은 값을 지정하는 대상이 될 수 없습니다.",
+        "booleanIsLowerCase": "변수 '{name}'은(는) 정의되지 않았습니다. '{booleanName}' 변수를 쓰려 했나요?",
```
Comment (microbit-matt-hillsdon) [inline packages/pyright-internal/src/localization/simplified.nls.ko.json]: I'll fix the quote inconsistency here

Comment (microbit-matt-hillsdon) [reply packages/pyright-internal/src/localization/simplified.nls.ko.json]: Fixed here: https://crowdin.com/translate/microbitorg/6424/en-ko#331204

Comment (microbit-matt-hillsdon) [reply packages/pyright-internal/src/localization/simplified.nls.ko.json]: Needs a sync to update this PR

**Pull request merged**